### DISCUSSION TITLE
In-game browser changelog page indentation failure

### DISCUSSION THREAD

**Discussion by Paladynee** [ppy/osu#17249] [Q&A] [answered]
![a](https://user-images.githubusercontent.com/62181440/158210753-e11ca482-2478-4e25-82a2-85a9cec03fd3.png)As you can see here, the in-game browser can not indent the \` marks into codeblocks.

![b](https://user-images.githubusercontent.com/62181440/158210950-0148ed4e-20e7-46ff-8007-0a97aed9cb9c.png)On the official page though, the indentation seems ok.

**Comment by Walavouchey** [comment, marked as answer]
see https://github.com/ppy/osu/issues/15730
